pkg/repositories/gormimpl/project_repo.go

```diff
@@ -70,3 +70,33 @@ func NewProjectRepo(db *gorm.DB, errorTransformer errors.ErrorTransformer,
 		metrics:          metrics,
 	}
 }
+
+func (r *ProjectRepo) UpdateProject(ctx context.Context, updatedProject models.Project) (models.Project, error) {
+	var project models.Project
+
+	timer := r.metrics.GetDuration.Start()
```

Comment:
ideally the get should be called at the manager layer

Reply:
Sounds goods.

Reply:
I'm following the structure of the `Get` operation defined in the same file here.

Reply:
sure but instead of duplicating the logic of the Get operation we could have the repo method caller (the project manager) call get and then update

Reply:
Got it, updated.